test/y2s390/zfcp_test.rb

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env rspec
+
+# Copyright (c) [2023] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../test_helper"
+require "y2s390/zfcp"
+
+describe Y2S390::ZFCP do
+  describe "#probe_controllers" do
+    before do
+      allow(Yast::SCR).to receive(:Read).with(Yast.path(".probe.storage"))
+        .and_return(controllers_data)
+
+      allow(Yast::SCR).to receive(:Execute).with(anything, /\/sbin\/vmcp q v fcp/)
+        .and_return(vmcp_output)
+
+      allow(Yast::SCR).to receive(:Execute).with(anything, /\/sbin\/cio_ignore -r/)
+        .and_return("exit" => 0, "stdout" => "")
+    end
+
+    let(:controllers_data) { load_data("probe_storage.yml") }
+
+    let(:vmcp_output) do
+      {
+        "exit"   => 0,
+        "stdout" => "FCP  F800 ON FCP   F807 CHPID 1C SUBCHANNEL = 000B\n" \
+                    "F800 TOKEN = 0000000362A42C00"
+      }
+    end
+
+    it "removes all FCP devices from the blacklist" do
```

Comment:
NP: FCP or zFCP?

Reply:
In this case, I would say FCP. Why? Because the code look for a column having that value. But do not ask me xD.